src/pages/Stock/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import ItemBlock from '@/components/ItemBlock';
+import mockData from '@/mock/product';
+import * as S from '@/pages/Vendor/Vendor.style';
+import { Title, Log } from '@/styles/common';
+
+const Stock = () => {
+  return (
+    <>
+      <S.VendorWrapper>
+        <S.ProductGrid>
+          {mockData.map(item => (
+            <ItemBlock
+              key={item.id}
+              name={item.name}
+              price={item.price}
+              stock={item.stock}
+              categoryId={item.categoryId}
+            />
+          ))}
```

Comment:
destructuring 하는 게 오히려 지저분해보이나요..? 밀리의 판단에 맡기겠습니다. 😅

```suggestion
          {mockData.map(({ id, name, price, stock, categoryId })=> (
            <ItemBlock
              key={id}
              name={name}
              price={price}
              stock={stock}
              categoryId={categoryId}
            />
          ))}
```

Reply:
destructuring이 더 깔끔해 보여요! 수정해 보겠습니다~